api/v1alpha1/groupversion_info.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Hard to tell if this whitespace correction was intentional, I have accidentally modified the license header a few times in the past so just wanted to point it out in case.